proposals/lip-0058.md

```diff
@@ -268,9 +292,29 @@ bftVotesSchema = {
   * `minActiveHeight`: The height from which the validator has been continuously active.
   * `largestHeightPrecommit`: The largest height for which a block generated by the validator implied a precommit.
 
-### Commands
+#### Genesis Data Substore
 
-The BFT module does not define any commands.
+##### Substore prefix, Store Key, and Store Value
+
+* The substore prefix is set to `SUBSTORE_PREFIX_GENESIS_DATA`.
+* The store key is set to empty bytes.
+* The store value is set to a data structure holding the timestamp of the genesis block of the chain, serialized using the `genesisDataSchema` schema presented below.
+* Notation: For the rest of this proposal let `genesisData` be the entry in the genesis data substore.
```

Comment:
you are a bit inconsistent with the capitalization of substore names in the LIP

Reply:
Changed to "Genesis Data substore" here and in other places in LIP 58 for consistency.

Reply:
I think you still have inconsistencies in the LIP

Reply:
I was looking at the outdated version, resolving.